rubicon/client/dataframe.py

```diff
@@ -38,9 +41,36 @@ def _get_data(self):
             project_name, self.id, experiment_id=experiment_id
         )
 
-    def plot(self, kind="table", **kwargs):
-        """Render the dataframe."""
-        raise NotImplementedError
+    def plot(self, **kwargs):
+        """Render the dataframe using `hvplot`.
+
+        Parameters
+        ----------
+        kwargs : dict
+            Additional keyword arguments to be passed along to the
+            `hvplot` function.
+
+        Notes
+        -----
+        For usage, visit: https://hvplot.holoviz.org/user_guide/Plotting.html
+        For customizations, visit:
+        https://hvplot.holoviz.org/user_guide/Customization.html
+
+        Examples
+        --------
+        >>> # Log a line plot
+        >>> dataframe.plot(kind='line', x='Year', y='Number of Subscriptions')
+        """
+        # data is a dask dataframe
+        if sys.modules.get("dask") is not None:
```

Comment:
when would this ever be `None`?

Reply:
i added it as a precaution -- only if we refactored to select either `pandas` or `dask` on install, both are currently installed as part of the dependencies now